code/__DEFINES/DNA.dm

```diff
@@ -235,16 +201,16 @@ GLOBAL_LIST_INIT(organ_process_order, list(
 #define SPECIES_GOLEM_SAND "sand_golem"
 #define SPECIES_GOLEM_GLASS "glass_golem"
 #define SPECIES_GOLEM_BLUESPACE "bluespace_golem"
-#define SPECIES_GOLEM_BANANIUM "bananium_golem"
+#define SPECIES_GOLEM_BANANIUM "ba_golem"
```

Comment:
Wtf? Undo all of these, you just made these worse for no gain

Reply:
they need to match icon states in human_parts.dmi and human_parts_greyscale.dmi